testonly/hammer/hammer.go

```diff
@@ -516,28 +530,27 @@ func (s *hammerState) getLeaves(ctx context.Context) error {
 }
 
 func (s *hammerState) getLeavesInvalid(ctx context.Context) error {
-	// TODO(phad): RevIsNegative ought to be invalid too but is accepted atm.
 	choices := []Choice{MalformedKey, RevTooBig, RevIsZero}
 
 	req := trillian.GetMapLeavesRequest{MapId: s.cfg.MapID}
+	rev := int64(-1)
```

Comment:
maybe add a constant at the top:
```go
// Requesting revision -1 in GetLeaves returns the most recent revision
const requestLatestRev = int64(-1)
```
('cos I keep forgetting about that aspect of the API...)